### PR TITLE
sl-144: Add show password button

### PR DESCRIPTION
**Pull request by jaroslawtt**
https://user-images.githubusercontent.com/87016333/229312040-813ecc33-b621-4dcc-a4fb-0c6288b64c95.mp4

### CODE REVIEW

File: frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx [hunk 1]
```diff
@@ -35,34 +44,62 @@ const CreatePasswordForm: React.FC<Properties> = ({
...
+            <Input
+              type={showNewPassword ? 'text' : 'password'}
+              label="New password"
+              name="newPassword"
+              control={control}
+              errors={errors}
+            />
+            <IconButton
+              label="New password"
+              icon="eye"
```
Comment (andanf-e) [inline frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: I think there should be 2 icons - to describe different states - one two show password (non-crossed eye), another for hide password (crossed eye)

File: frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx [hunk 2]
```diff
@@ -35,34 +44,62 @@ const CreatePasswordForm: React.FC<Properties> = ({
...
   );
 
+  const [showPassword, toggleShowPassword] = useToggle(false);
+  const [showNewPassword, toggleShowNewPassword] = useToggle(false);
+  const [showRepeatPassword, toggleShowRepeatPassword] = useToggle(false);
+
   return (
     <Modal isOpen={isOpen} onClose={onCloseModal}>
-      <div className={getValidClassNames(styles['form-wrapper'], className)}>
+      <div className={getValidClassNames(styles[inputWrapper], className)}>
```
Comment (andanf-e) [inline frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: Let's get rid of constants for class names, cause in some places you're using it and in other places do not use,, and when I am looking on it - I need to search for a place where these constants are defined, and try to understand on which styles file in relies
```suggestion
      <div className={getValidClassNames(styles['input-wrapper'], className)}>
```

Comment (jaroslawtt) [reply frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: lint error in that case

Comment (what1s1ove) [reply frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: share the screen with error please

Comment (jaroslawtt) [reply frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: image

Comment (what1s1ove) [reply frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: just give a different class names

Comment (jaroslawtt) [reply frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx]: even if it shares the same styles?

**Comment by jaroslawtt** [comment]
https://user-images.githubusercontent.com/87016333/229346750-b5fd145e-9077-468d-87c5-1ccd3e7fde10.mp4

**Review by Izdormu — APPROVED** [review]

File: frontend/src/libs/hooks/use-toggle/use-toggle.hook.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { useCallback, useState } from '~/libs/hooks/hooks.js';
+
+type ReturnValue = [value: boolean, toggleValue: () => void];
+
+const useToggle = (initialValue: boolean): ReturnValue => {
+  const [value, setValue] = useState(initialValue);
+
+  const toggleValue = useCallback(() => setValue(!value), [value, setValue]);
```
Comment (what1s1ove) [inline frontend/src/libs/hooks/use-toggle/use-toggle.hook.ts]:
```suggestion
  const handleValueToggle = useCallback(() => setValue(!value), [value, setValue]);
```

File: frontend/src/libs/hooks/use-toggle/use-toggle.hook.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { useCallback, useState } from '~/libs/hooks/hooks.js';
+
+type ReturnValue = [value: boolean, toggleValue: () => void];
+
+const useToggle = (initialValue: boolean): ReturnValue => {
+  const [value, setValue] = useState(initialValue);
```
Comment (what1s1ove) [inline frontend/src/libs/hooks/use-toggle/use-toggle.hook.ts]: type state

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by Izdormu — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**